CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- Removed deprecated code (#764)
+
```

Comment:
In conf.py I see the following deprecated stuff in ignore_paths:

'../qtt/deprecated/.*.py', '../qtt/deprecated/tunnelbarrier.py'

the latter one is enclosed in the first one and this file tunnelbarrier.py doesn't exist anymore.